modules/hostap/src/supp_api.c

```diff
@@ -430,6 +430,105 @@ static int wpas_config_process_blob(struct wpa_config *config, char *name, uint8
 
 	return 0;
 }
+
+static const struct wifi_cipher_desc ciphers[] = {
+	{WPA_CAPA_ENC_CCMP_256, "CCMP-256"}, {WPA_CAPA_ENC_GCMP_256, "GCMP-256"},
+	{WPA_CAPA_ENC_CCMP, "CCMP"},         {WPA_CAPA_ENC_GCMP, "GCMP"},
+	{WPA_CAPA_ENC_TKIP, "TKIP"},         {WPA_CAPA_ENC_WEP104, "WEP104"},
+	{WPA_CAPA_ENC_WEP40, "WEP40"}};
```

Comment:
These look bad, could you align them and put one line in a row. So
```
	{WPA_CAPA_ENC_CCMP_256, "CCMP-256"},
	{WPA_CAPA_ENC_GCMP_256, "GCMP-256"},
	{WPA_CAPA_ENC_CCMP, "CCMP"},
	{WPA_CAPA_ENC_GCMP, "GCMP"},
	{WPA_CAPA_ENC_TKIP, "TKIP"},
	{WPA_CAPA_ENC_WEP104, "WEP104"},
	{WPA_CAPA_ENC_WEP40, "WEP40"},
};
```

Reply:
Updated as comment.